### ISSUE TITLE
Problem loading Media Uploader after importing

### ISSUE THREAD

**Issue by kokuou** [manovotny/wptest#44]
I started with a clean database and re-installed my theme, but after importing the test data, the Media Uploader no longer works when trying to view media from All Dates. Every folder will display except for July 2011. I've tried waiting several minutes, but it shouldn't take that long?

I'm developing locally on MAMP, if that matters any.

**Comment by manovotny** [comment]
@kokuou : : 

Hrm, no. It shouldn't take that long.

If you look under `wp-content/uploads/2011/07`, do you actually see any of the assets (aka. images and audio files)?

**Comment by kokuou** [comment]
Yeah, it looks like it's importing all the files properly (that folder exists and has images) but when I have 'All Dates' or 'July 2011' selected in the Media Uploader, it just never shows anything. To be safe, I left it going overnight, and it still didn't show by this morning.

**Comment by manovotny** [comment]
Cool. Thanks. That eliminates that.

I am a MAMP user too, so let's see if we can recreate this...

What are you using for the following.

1. What browser and version?
2. What version of PHP in MAMP?
3. What version of WordPress?
4. Any plugins activated in WordPress?

**Comment by kokuou** [comment]
No problem! Other than the Media Uploader issue, it's a fantastic resource for development, so I'm happy to help! :)

1) Browser is Chrome, latest version (42.0.2311.90 m)
2) MAMP PHP version is 5.6.3
3) WordPress version is the latest (4.2)
4) Plugins are as follows:
  - Akisment (there by default, unactivated)
  - Hello Dolly (same)
  - Theme Check (activated)
  - WordPress Database Reset (unactivated, but the same problem occurred previously with the same folder when this plugin was not installed)
  - WordPress Importer (activated)

That's it. Anything else I should check?

**Comment by kokuou** [comment]
Okay, an update. I just did a completely fresh install of WP in a different directory and imported the data, and it seems that all is working, so it looks like it's something on my end. I'll try and troubleshoot and report back if I find anything.

**Comment by kim07** [comment]
thanks got it fixed :)
